hls4ml/writer/quartus_writer.py

```diff
@@ -575,20 +580,23 @@ def write_activation_tables(self, model):
             in_val = 2*8.0*(i-float(table_size)/2.0)/float(table_size)
```

Comment:
Table range is now hardcoded to be from -8 to 8, and for `tanh` the range is from -4 to 4

Reply:
This code was not modified by me and is working , we will not fully rewrite hls4ml